src/components/WorkflowForms/NewClientSignup/IntroAndInitialAdvice/IntroAndInitialAdvice.jsx

```
@@ -0,0 +1,47 @@
+  const language = useContext(LanguageContext);
+  strings.setLanguage(language || mva_utils.get_device_language());
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleNext = async () => {
+    setIsLoading(true);
+    try {
+      await api.complete_workflow_step({
+        workflow_business_key: caseId,
+        task_definition_key: "introAndInitialAdvice"
+      });
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(no-unused-vars)